mkdocs.yml

```diff
@@ -0,0 +1,32 @@
+# MkDocs options.
+# See: https://www.mkdocs.org/user-guide/configuration/
+site_name: Coda Packs SDK
+repo_url: https://github.com/coda/packs-sdk
+docs_dir: docs
+markdown_extensions:
+  - admonition
+  - attr_list
+  - pymdownx.details
+  - pymdownx.highlight:
+      linenums: true
+  - pymdownx.keys
+  # Snippets inclusion.
+  # See: https://facelessuser.github.io/pymdown-extensions/extensions/snippets/
+  - pymdownx.snippets:
+      base_path: documentation/examples
```

Comment:
What exactly does this do with the example snippets?

Reply:
Nothing as per this PR, but in my demo I was using it to pull the IDE snippets into guides, to single-source some of the content. I can strip this file down for the initial commit and then add to it we build out the docs if you'd prefer.

Reply:
Yeah let's just tackle that separately.

Reply:
Done